src/app/components/CourseChoiceButton/CourseChoiceButton.module.css

```diff
@@ -0,0 +1,10 @@
+.button {
+  color: var(--contrast-text-color);
+  background: var(--primary-color);
+  font-size: 1.25em;
+  font-weight: bold;
+  border: none;
+  border-radius: 10px;
```

Comment:
maybe use em for border-radius

Reply:
I tried. Since it is an oval element the border radius makes it an oval when using em.